__tests__/__snapshots__/packageData.test.js.snap

```diff
@@ -241,6 +241,7 @@ Object {
   "@babel/template": "7.0.0-beta.39",
   "@babel/traverse": "7.0.0-beta.39",
   "@babel/types": "7.0.0-beta.39",
+  "[object Object]": "7.0.0-beta.39",
```

Comment:
@hzoo hmm... It seems better to expand it as a character string?  But maybe it is difficult.🤔

```
expect(upgradeDeps(depsFixture, VERSION)).toMatchSnapshot();
```